### ISSUE TITLE
[BUG] The page is not completely responsive

### ISSUE THREAD

**Issue by GedeonTS** [ever-co/ever-teams#2432]
![Screenshot from 2024-04-25 16-27-46](https://github.com/ever-co/ever-teams/assets/97834160/682aa09b-3a28-4d42-b179-386b958fd12f)
**Description**: on the team dashboard when we are displaying the member in cards or table mode the responsiveness of the page breaks between 575px and 800px 

**Steps to Reproduce**:
Add at least one team member to the current team, then use a device having between 575px and 800px screen width or inspect with your browser's devTools to see the responsiveness

**Current Behaviour**:
The timezone field is not aligned horizontally, which decreases the user-experience.

**Expected Behaviour**:
The page should be completely responsive for an optimal experience for all the users..

**Comment by Anishali2** [comment]
I don't think that in this project every screen is 500px responsive except the login page or some other pages but on the dashboard side it's not fully responsive.
Our main priority first is functionality + performance + design.

@evereq Sir please guide about responsiveness.

Here is the list of current designs in mobile responsiveness.

![image](https://github.com/ever-co/ever-teams/assets/105014311/1063ab11-125f-4985-bbc3-4c3afe4190ac)

![image](https://github.com/ever-co/ever-teams/assets/105014311/3505c0fd-fd39-4282-9724-379878369ae0)

![image](https://github.com/ever-co/ever-teams/assets/105014311/c0d9bfc1-e270-470a-b6b7-b82bb46133c3)

![image](https://github.com/ever-co/ever-teams/assets/105014311/24cec619-2cb4-4579-8139-e33ee676f8d0)

**Comment by Anishali2** [comment]
Also if we want to make this section response this should be cards view in small screen because there is not much space here to fit those thing until unless we use the horizontal scrollbar

**Comment by evereq** [comment]
100% agree with @Anishali2 , we will work on responsiveness separately as it's require a lot of design decisions. Let's keep this ticket as a starting point for future discussion.

**Comment by Tacite243** [comment]
Hey! I would like to work on this task, could you please assign it to me?

**Comment by evereq** [comment]
@Tacite243 sure, it will be great if you can help with this. Basically some things should be easy to make responsive, while others will require redesign specially for mobile / tablet screens etc. So if you can handle more simple changes first, that would be great start. Suggest to create separate small PRs for each improvement if possible, don't do too many in one PR.

**Comment by Tacite243** [comment]
Sure, I'll get to it!
